### ISSUE TITLE
GENIE v3_02_00 libraries not found

### ISSUE THREAD

**Issue by ikatza** [NuSoftHEP/nusimdata#4]
The GENIE Glashow resonance libraries are not found on cvmfs, so trying this:

```
ls /cvmfs/larsoft.opensciencegrid.org/products/genie/v3_02_00/Linux64bit+3.10-2.17-*/lib/ | grep GPhGlwResXS
```
returns empty. It's OK if instead of `v3_02_00` I put `v3_00_06p`.

This might no be the right repository to open this issue, I don't know if the problem is that these libs don't exist anymore or have been renamed. Or that for some reason, they weren't published to cvmfs. 

If the problem is that these libraries do not longer exit in GENIE v3_02, the offending line is:
https://github.com/NuSoftHEP/nusimdata/blob/005adc473fd7e5156fd7bde3b9585bdf6150755f/Modules/FindGENIE.cmake#L61

**Comment by knoepfel** [comment]
@ikatza, GENIE 3.02 does have some changes to the libraries, and our build of GENIE 3.02.00 had some errors in the `FindGENIE.cmake` module.  However, we have fixed that according to Robert Hatcher's suggestion--we unfortunately do not yet have a release of that available:

https://github.com/NuSoftHEP/nusimdata/blob/v3_12_03_01_br/Modules/FindGENIE.cmake

However, our corrected CMake module does not have the `GPhGlwResXS` CMake target either for GENIE 3.02.  @nusense, can you comment whether the `GPhGlwResXS` library should be present for GENIE 3.02?

**Comment by nusense** [comment]
No, the GPhGlwResXS is among the libraries that were replaced by others.   That one in particular was replaced by, I believe, GPhHELptnXS.

But the list of libraries for GENIE v3.02 is in the FindGENIE.cmake file.   Eventually this list should be derived from ` genie-config —libs` but how to do that via  CMake at run/build time isn’t yet clear, so for now we explicitly list all the libraries we expect to be built as part of the GENIE installation.

-Robert

On Sep 15, 2022, at 8:19 AM, Kyle Knoepfel ***@***.******@***.***>> wrote:



@ikatza<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_ikatza&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=m-HXyHJ8kCwT6sbzoSlYdcpu3Z4Noqd50sLaf0Gm5RY&m=7FEVTpJGtKmRAC-RXAI9EomO4gGqL5D9__IKgkHEeNlS0UeIzwDagHF7pl1Qudwu&s=GhTlQH0j7aHYMqGSHdFGjpEMMiqTGy5nskLNEXzNuCU&e=>, GENIE 3.02 does have some changes to the libraries, and our build of GENIE 3.02.00 had some errors in the FindGENIE.cmake module. However, we have fixed that according to Robert Hatcher's suggestion--we unfortunately do not yet have a release of that available:

https://github.com/NuSoftHEP/nusimdata/blob/v3_12_03_01_br/Modules/FindGENIE.cmake<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_NuSoftHEP_nusimdata_blob_v3-5F12-5F03-5F01-5Fbr_Modules_FindGENIE.cmake&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=m-HXyHJ8kCwT6sbzoSlYdcpu3Z4Noqd50sLaf0Gm5RY&m=7FEVTpJGtKmRAC-RXAI9EomO4gGqL5D9__IKgkHEeNlS0UeIzwDagHF7pl1Qudwu&s=1kym-UK7CDVWLBz0cGkBoTyzLFgMnQtooS9-HnaafAA&e=>

However, our corrected CMake module does not have the GPhGlwResXS CMake target either for GENIE 3.02. @nusense<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_nusense&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=m-HXyHJ8kCwT6sbzoSlYdcpu3Z4Noqd50sLaf0Gm5RY&m=7FEVTpJGtKmRAC-RXAI9EomO4gGqL5D9__IKgkHEeNlS0UeIzwDagHF7pl1Qudwu&s=GXkOsNos57Kz_twsumDcxqmO3Xfv4ONZwwF8y9O0EHA&e=>, can you comment whether the GPhGlwResXS library should be present for GENIE 3.02?

—
Reply to this email directly, view it on GitHub<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_NuSoftHEP_nusimdata_issues_4-23issuecomment-2D1248091238&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=m-HXyHJ8kCwT6sbzoSlYdcpu3Z4Noqd50sLaf0Gm5RY&m=7FEVTpJGtKmRAC-RXAI9EomO4gGqL5D9__IKgkHEeNlS0UeIzwDagHF7pl1Qudwu&s=qhCn_bZnfn32CmY9_HTs1jVT0TJfSUIHuO50EhUZY0I&e=>, or unsubscribe<https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ABUURYXUZQKZFUHLZYSKVR3V6MO43ANCNFSM6AAAAAAQNILEH4&d=DwMCaQ&c=gRgGjJ3BkIsb5y6s49QqsA&r=m-HXyHJ8kCwT6sbzoSlYdcpu3Z4Noqd50sLaf0Gm5RY&m=7FEVTpJGtKmRAC-RXAI9EomO4gGqL5D9__IKgkHEeNlS0UeIzwDagHF7pl1Qudwu&s=72M3lc61DH6l2Nt4O6RV27XGZVLMkOQgPR3OSaHKGTc&e=>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

Robert Hatcher
Computational Physics Developer

Scientific Computing Division
Fermi National Accelerator Laboratory
PO Box 500, MS 234, Batavia, IL 60510, USA
***@***.******@***.***>
Office: 630-840-3102<tel:630-840-3102>
Cell:    630-234-0091<tel:630-234-0091>
